src/Verilator.cpp

```diff
@@ -260,6 +260,7 @@ static void process() {
             // Remove cell arrays (must be between V3Width and scoping)
             V3Inst::dearrayAll(v3Global.rootp());
             V3LinkDot::linkDotArrayed(v3Global.rootp());
+            V3Error::abortIfErrors();
```

Comment:
Why is this needed?  Means the user will see fewer errors they can fix in a single run.